src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import memesData from "./memesData.js";
```

Comment:
One of the feature for meme generator is to get the memes images from an API. Could you implement that functionality instead of getting the memes from a local array